test/index.js

```diff
@@ -460,4 +470,99 @@ describe('KeyringController', function () {
       );
     });
   });
+
+  describe('cacheEncryptionKey', function () {
+    it('sets encryption key data upon submitPassword', async function () {
+      keyringController.cacheEncryptionKey = true;
+      await keyringController.submitPassword(password);
+
+      expect(keyringController.password).toBe(password);
+      expect(keyringController.memStore.getState().encryptionSalt).toBe('SALT');
+      // eslint-disable-next-line jest/no-restricted-matchers
```

Comment:
Nit: In these cases where w want to ensure we have a non-empty string, we can use `expect(...).toStrictEqual(expect.stringMatching('.+'))`. This is a bit more specific than the truthy check in that it ensures it's a string, and it avoids the need for this ignore comment.

Reply:
Excellent, I had no idea!

Reply:
There is one more place you could use this (on this line :sweat_smile: )